buildscripts/update-lobster.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+set -o errexit
+set -o xtrace
+
+SCRIPTS_DIR=buildscripts
+LOBSTER_REPO=https://github.com/evergreen-ci/lobster.git
+LOBSTER_DIR=.lobster-temp
+LOBSTER_ASSETS_DIR=build
+LOBSTER_STATIC_DIR=static
+LOBSTER_HTML=index.html
+EVG_LOB_STATIC_DIR=public/static/lobster/static
+EVG_LOB_MANIFEST_DIR=public/static/lobster
+EVERGREEN_JS_DIR=js
+EVERGREEN_CSS_DIR=css
+EVERGREEN_TEMPLATES_DIR=service/templates
+
+# clone lobster repo
+pushd $SCRIPTS_DIR
+rm -rf $LOBSTER_DIR
```

Comment:
Remove the `rm -rf`s and just put things in the bin directory for temporary build artifacts.